src/content/docs/en/pages/architectures/edge-enhanced/edge-enhanced.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Enhance Applications with Edge Computing
+description: Learn how to enhance your applications and APIs using Edge Computing.
+meta_tags: edge, applications, web applications, api, enhancement
+namespace: docs_arch_edge_enhancement
+permalink: /documentation/architectures/edge-enhanced/
+---
+
+Edge computing is a technology that brings data storage and processing closer to the source of data - the edge of your network. The use of edge computing introduces several advantages, mainly reduced latency, increased security, and improved scalability. 
+
+Using Azion services and products, we demonstrate how an ecommerce application's shopping cart feature can be moved to an edge computing environment. The proposed serverless approach integrated with other edge capabilities offers high performance, scalability, and reliable data transaction.
+
+## Cart Management at the Edge
+
+By running the shopping cart at the edge using edge functions, companies can reap several benefits:
+
+- **Reduced Latency**: since the shopping cart operations are performed closer to the user, the response times are significantly reduced, leading to a smoother and faster user experience.
```

Comment:
```suggestion
- **Reduced latency**: since the shopping cart operations are performed closer to the user, the response times are significantly reduced, leading to a smoother and faster user experience.
```